state-manager-frontend/src/components/NodeSchemaViewer.tsx

```
@@ -0,0 +1,149 @@
+
+export const NodeSchemaViewer: React.FC<NodeSchemaViewerProps> = ({
+  node,
+  isExpanded = false,
+  onToggle
+}) => {
+  return (
+    <div className="bg-white rounded-lg shadow-md border border-gray-200 overflow-hidden">
+      <div 
+                    className="bg-gradient-to-r from-[#031035] to-purple-600 p-4 text-white cursor-pointer hover:from-[#0a1a4a] hover:to-purple-700 transition-all duration-200"
+        onClick={onToggle}
+      >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)